projects/plugins/boost/app/assets/src/js/stores/config.ts

```diff
@@ -22,6 +23,20 @@ export const dismissedPopOuts = {
 	},
 };
 
+// eslint-disable-next-line camelcase
+const getStartedStore = writable( Jetpack_Boost.site.getStarted );
+export const getStarted = {
+	subscribe: getStartedStore.subscribe,
+	setValue: async ( value: boolean ) => {
```

Comment:
This method has no reason to be async. I can see it's called from `onConnectionComplete` without an await.

I also think this store has a slightly confusing API. `setValue` forces a flag in local memory which doesn't get pushed to the remote host. `done` sends the updated value to the back-end. I feel like this will lead to confusion in the future.

Reply:
I have removed the separate store and replaced it with usage from directly the config store to make it easier to refresh and use a single source of truth.

Reply:
👍 Nice!